alphafold3_pytorch/attention.py

```diff
@@ -224,11 +223,7 @@ def __init__(
         self.to_gates = None
 
         if gate_output:
-            gate_linear = nn.Linear(dim, dim_inner)
-            nn.init.zeros_(gate_linear.weight)
-            nn.init.constant_(gate_linear.bias, init_gate_bias)
-
-            self.to_gates = gate_linear
+            self.to_gates = nn.Sequential(nn.Linear(dim, dim_inner, bias = False), nn.Sigmoid())
```

Comment:
want to use the LinearNoBias here?

Reply:
Sure

Reply:
Had to duplicate to avoid circular imports

Reply:
yeah that's fine, dryness is overrated